__tests__/src/index.spec.ts

```diff
@@ -395,7 +441,7 @@ describe("flatpickr", () => {
 
         fp.setDate("Thu 20 October, '16");
         expect(fp.input.value).toEqual(RESULT);
-        fp.setDate("2015-11-21 19:29:49");
+        fp.setDate("2015-11-21T19:29:49Z");
```

Comment:
This needed to be changed because of the change in Regex resulted in producing a warning like this: `Error: Invalid date provided: 2015-11-21 19:29:49`.